src/apis/edit.rs

```diff
@@ -25,9 +25,42 @@ pub struct EditResponse {
 }
 
 pub async fn edit(
-    req: actix_web::web::Json<EditRequest>,
+    req: actix_web::HttpRequest,
+    req_data: actix_web::web::Json<EditRequest>,
+    data: actix_web::web::Data<fpm::commands::serve::AppState>,
 ) -> actix_web::Result<actix_web::HttpResponse> {
-    match edit_worker(req.0).await {
+    let mut config = match fpm::Config::read(None, false).await {
```

Comment:
Not fond of this `read()` invocation. If we are often going to call read with None and false, we should have another `read` method. Why are we passing these parameters to read at all if they are not needed for proper functioning.

Reply:
We will do it later while we refactor the code.